source/perflib/perflib.h

```diff
@@ -41,6 +41,10 @@ int perflib_create_ssl_ctx_pair(const SSL_METHOD *sm, const SSL_METHOD *cm,
                                 int min_proto_version, int max_proto_version,
                                 SSL_CTX **sctx, SSL_CTX **cctx, char *certfile,
                                 char *privkeyfile);
+int perflib_create_ossl_lib_ctx_pair(OSSL_LIB_CTX *libctx, const SSL_METHOD *sm,
```

Comment:
What is the reason to introduce `perflib_create_ossl_lib_ctx_pair()`? I think the `libctx` argument can be just added to existing function `perflib_create_ssl_objects()`. Caller can then pass either NULL or pointer to libctx.

the change looks good otherwise. this is the only question I have.

thanks.